testing/setup/setup.sh

```diff
@@ -0,0 +1,199 @@
+# Environment Variables Config
+export CLUSTER_NAME=testing-ci-cluster-2
+export GUEST_CLUSTER_NAME=example-guest-cluster
+export AWS_PROFILE=karpenter-ci
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export AWS_REGION=us-west-2
+export KARPENTER_VERSION=v0.10.1
+
+# Create Cluster
+cat <<EOF > cluster.yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_REGION}
+  version: "1.21"
+  tags:
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+managedNodeGroups:
+  - instanceType: m5.large
+    amiFamily: AmazonLinux2
+    name: ${CLUSTER_NAME}-ng
+    desiredCapacity: 1
+    minSize: 1
+    maxSize: 10
+iam:
+  withOIDC: true
+EOF
+eksctl create cluster -f cluster.yaml
+#----------------
+
+# Tekton - Do one at a time
```

Comment:
Make sure this script is executable. Insert sleeps if you must.

Reply:
Good point.